dev/com.ibm.ws.session.cache/resources/com/ibm/ws/session/store/cache/resources/WASSessionCache.nlsprops

```diff
@@ -19,4 +19,32 @@
 
 INCORRECT_URI_SYNTAX=SESN0300E: Invalid syntax in httpSessionCache URI.  Cause is {0}.
 INCORRECT_URI_SYNTAX.explanation=The syntax of the specified httpSessionCache JCache configuration URI is invalid.
-INCORRECT_URI_SYNTAX.useraction=Correct the syntax of the specified httpSessionCache configuration URI.
\ No newline at end of file
+INCORRECT_URI_SYNTAX.useraction=Correct the syntax of the specified httpSessionCache configuration URI.
+
+ERROR_REMOVING_SESSION=SESN0301E: An error occurred when trying to remove a session from the cache. The exception is: {0}.
+ERROR_REMOVING_SESSION.explanation=An error occurred when trying to remove a session from the cache. See exception for further details.
+ERROR_REMOVING_SESSION.useraction=See the exception for more details. The JCache provider's configuration may need to be updated.
+
+PROP_HIT_ERROR=SESN0302E: An exception occurred when storing application data changes to the cache. The exception is: {0}.
+PROP_HIT_ERROR.explanation=An exception occurred when storing application data changes to the cache. See exception for further details.
+PROP_HIT_ERROR.useraction=See the exception for more details. The JCache provider's configuration may need to be updated.
+
+STORE_SESS_ERROR=SESN0303E: An exception occurred when trying to store a session in the cache. The exception is: {0}.
```

Comment:
Why do some of these messages use "... when trying to do X" and others use the form "... when doing X" ?  Can we make them all consistent.  The latter, which is more concise, seems preferable.